packages/assets/scss/03-organisms/_mass-feedback-form.scss

```diff
@@ -58,10 +58,11 @@ $formQuickTransitions: max-height 0.3s linear;
 
 
     max-height: 0;
-    overflow: hidden;
+    display: none;
```

Comment:
@arthurbaghdas This change is not appearing in the `openmass` PR.  I see you changed its Mayflower version pointing to this branch in it though.  The hidden button is still focusable.
Also, as you add `display: none;`, remove L. 60 `max-height: 0;`.

Reply:
Ok